src/components/item/createItem/createItemComp.css

```diff
@@ -155,36 +172,62 @@
 }
 
 .createItemComp-layout .createItemComp-create-btn{
-    padding: 1em 2m 1em 2em;
-    background: linear-gradient(180deg, #79C652 0%, #325F50 100%);
-    border-radius: 25.5px;
-    color: white;
-    border-color: white;
-    border-width: 1px;
+    padding: 0.5em 2em 0.5em 2em;
+    background: #6FF67E;
+    color: black;
+    font-weight: 500;
+    border: 0;
+    margin-top: 1em;
+
 }
 
 .createItemComp-layout .createItemComp-image{
     border-radius: 10px;
     width: 260px;
     height: 260px;
+    margin-top: 3em;
 }
 
+.preview-image-box .preview-image1 {
+    width: 130px;
+    height: 130px;
+}
 .createItemComp-layout .createAuction-method{
     background-color: transparent;
-    color: #EBEBEB;
```

Comment:
@rainbow-catch plz revert this color.
The original one is better

Reply:
Got it